README.md

```diff
@@ -46,34 +54,44 @@
 
 ### Documentation
 
-- Installation, Configuration, and usage documentation available at [Read The Docs](https://neo-python.readthedocs.io/en/latest/)
+The full documentation on how to install, configure and use neo-python can be
+found at [Read The Docs](https://neo-python.readthedocs.io/en/latest/).
 
-### Get Help or give help
+### Get help or give help
 
 - Open a new [issue](https://github.com/CityOfZion/neo-python/issues/new) if you encounter a problem.
 - Or ping **@localhuman**  or **@metachris** on the [NEO Discord](https://discord.gg/R8v48YA).
-- Pull requests welcome. You can help with wallet functionality, writing tests or documentation, or on any other feature you deem awesome. All successful pull requests will be rewarded with one photo of a cat or kitten.
-
+- Pull requests welcome. You can help with wallet functionality, writing tests
+  or documentation, or on any other feature you deem awesome. All successful
+  pull requests will be rewarded with one photo of a cat or kitten.
 
 ## Getting started
 
-You will need to install the libleveldb library. Install [Python 3.5](https://www.python.org/downloads/release/python-354/) to make sure you don't run into any issues with your version of Python being different than the current maintainer's version. Note that Python 3.6 is not currently supported due to incompatibilities with the byteplay module.
+neo-python has two System dependencies (everything else is covered with `pip`):
 
-We have published a Youtube [video](https://youtu.be/oy6Z_zd42-4) to help get you started with this library. There are other videos under the [CityOfZion](https://www.youtube.com/channel/UCzlQUNLrRa8qJkz40G91iJg) Youtube channel.
+- LevelDB
+- Python **3.5** (3.6 and up are currently not supported)
 
-##### OSX:
+### LevelDB
+
+We have published a Youtube [video](https://youtu.be/oy6Z_zd42-4) to help get
```

Comment:
the videos should be where they were in "Getting started" and not in LevelDB

Reply:
The mentioning of `to help get you started with this library` (I thought it refers to LevelDB) was confusing me towards that interpretation, please check whether it fits now.